src/elements/content-sidebar/activity-feed/annotations/AnnotationActivityItem.js

```diff
@@ -0,0 +1,28 @@
+// @flow
+import * as React from 'react';
+import noop from 'lodash/noop';
+import { FormattedMessage, type MessageDescriptor } from 'react-intl';
+import NavButton from '../../../common/nav-button';
+import { Link } from '../../../../components/link';
+
+type Props = {
+    href?: string,
+    id: string,
+    message: MessageDescriptor,
+    onClick: (id: string) => void,
+};
+
+const AnnotationActivityItem = ({ href, id, message, onClick = noop }: Props): React.Node => {
+    const handleClick = event => {
+        event.preventDefault();
+
+        onClick(id);
+    };
+    return (
+        <NavButton component={Link} className="bcs-AnnotationActivity-link" onClick={handleClick} href={href}>
```

Comment:
Does this need to be a `NavButton`? Seems like more of a plain button  with an onClick